kernel/packages/entryPoints/website.ts

```diff
@@ -44,25 +44,19 @@ import { filterInvalidNameCharacters, isBadWord } from 'shared/profiles/utils/na
 const logger = createLogger('website.ts: ')
 
 function configureTaskbarDependentHUD(i: UnityInterface, voiceChatEnabled: boolean) {
-  i.ConfigureHUDElement(HUDElementID.TASKBAR, { active: true, visible: true }, { enableVoiceChat: voiceChatEnabled })
+  i.ConfigureHUDElement(
+    HUDElementID.TASKBAR,
+    { active: true, visible: true },
+    {
+      enableVoiceChat: voiceChatEnabled
+    }
+  )
   i.ConfigureHUDElement(HUDElementID.WORLD_CHAT_WINDOW, { active: true, visible: true })
 
   i.ConfigureHUDElement(HUDElementID.CONTROLS_HUD, { active: true, visible: false })
   i.ConfigureHUDElement(HUDElementID.EXPLORE_HUD, { active: true, visible: false })
   i.ConfigureHUDElement(HUDElementID.HELP_AND_SUPPORT_HUD, { active: true, visible: false })
 }
-/**
- * Subscribe to uncaught errors
- */
-window.addEventListener('error', (event: ErrorEvent) => {
-  ReportFatalError(event.message as any, {
-    type: event.type,
-    message: event.message,
-    stack: event.error.stack,
-    filename: event.filename
-  })
-  return false
-})
```

Comment:
@caminoezequiel This was causing all the errors to be reported twice and keeping the application blocked and unresponsive.

Reply:
The reportFatalError is properly called from `globalThis.UnityLoader.Error.handler`

Reply:
What about uncatched errors coming from the non-UnityLoader thing? (i.e. something related to the kernel?) Even if we have typescript type-safety, there is no assurance that inputs are completely sanitized.

Reply:
@menduz I added this listener to avoid duplicate error reporting.
I removed `reportFatalError` in `globalThis.UnityLoader.Error.handler` because it was the problem of duplicate errors also when `reortFatalError` was calling from `globalThis.UnityLoader.Error.handler` it wasn't reporting data to segment.